### PR TITLE
Added OneHot to raw_ops

### PR DESCRIPTION
**Pull request by KareemMAX**
Close https://github.com/unifyai/ivy/issues/5422

### CODE REVIEW

File: ivy/functional/frontends/tensorflow/raw_ops.py [hunk 1]
```diff
@@ -276,6 +276,13 @@ def NthElement(*, input, n, reverse=False, name="NthElement"):
     return ivy.astype(ivy.sort(input, descending=reverse)[..., n], input.dtype)
 
 
+@to_ivy_arrays_and_back
+def OneHot(*, indices, depth, on_value, off_value, axis=-1, name="OneHot"):
```
Comment (CatB1t) [inline ivy/functional/frontends/tensorflow/raw_ops.py:280]: We should flag that the function does only support few data types, `int32, int64, uint8`

**Review by CatB1t — CHANGES_REQUESTED** [review]

**Comment by KareemMAX** [comment]
Addressed your issues and resolved conflicts

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

File: ivy/functional/frontends/tensorflow/raw_ops.py [hunk 2]
```diff
@@ -276,6 +276,16 @@ def NthElement(*, input, n, reverse=False, name="NthElement"):
...
+    return ivy.one_hot(
+        indices, depth, on_value=on_value, off_value=off_value, axis=axis
+    )
+
+
+OneHot.supported_dtypes = ("int32", "int64", "uint8")
```
Comment (CatB1t) [inline ivy/functional/frontends/tensorflow/raw_ops.py:286]: we no longer use this, we now have a decorator for that: https://lets-unify.ai/ivy/deep_dive/data_types.html#supported-and-unsupported-data-types

**Review by CatB1t — CHANGES_REQUESTED** [review]

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**